src/topp/PeakPickerIM.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2002-present, The OpenMS Team -- EKU Tuebingen, ETH Zurich, and FU Berlin
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// --------------------------------------------------------------------------
+// $Author: Mohammed Alhigaylan $
+// $Maintainer: Timo Sachsenberg $
+// -------------------------------------------------------------------------------------------------------------------------------------------
+
+#include <OpenMS/CONCEPT/LogStream.h>
+#include <OpenMS/FORMAT/FileHandler.h>
+#include <OpenMS/FORMAT/MzMLFile.h>
+#include <OpenMS/KERNEL/MSExperiment.h>
+#include <OpenMS/APPLICATIONS/TOPPBase.h>
+#include <OpenMS/FORMAT/DATAACCESS/MSDataWritingConsumer.h>
+#include <OpenMS/PROCESSING/CENTROIDING/PeakPickerIM.h>
+
+using namespace OpenMS;
+using namespace std;
+
```

Comment:
Add a generic description of what the algorithm is doing, e.g. `Pick peaks in IM dimension by collapsing IM onto m/z ... etc etc`